mkdocs.yml

```diff
@@ -47,7 +47,8 @@ pages:
             - Source code : 'AutolabJS v0.2.0/v0.2.0-IO-Tests.md'
             - Compiled binaries : 'AutolabJS v0.2.0/v0.2.0-IO-Tests-Binaries.md'
         - Post lab self evaluation : 'AutolabJS v0.2.0/v0.2.0-Post-lab-self-evaluation.md'
-  - AutloabJS Coding Standards : 'AutolabJS-coding-standards.md'
+  - Product Features : 'Product-Features.md'
+  - AutolabJS Coding Standards : 'AutolabJS-coding-standards.md'
```

Comment:
This link can just be called coding standards in light of its aoolicavikity to all projects of AutolabJS